Makefile

```diff
@@ -1,5 +1,15 @@
+# Check for OS
+BUILD_CGO_ENABLED ?= 0
+UNAME_S := $(shell uname -s)
+ifeq ($(UNAME_S),Darwin)
+	shasum = shasum -a 512
+	BUILD_CGO_ENABLED ?= 1
```

Comment:
This won't work on Darwin, since `BUILD_CGO_ENABLED ?= 1` only will assign the variable if it is unsigned, which it will never be. 

I'll move row 2 to line 9 to keep the variable over-writable by command line, if one for some reason would like that.